crates/wash/tests/parser/main.rs

```diff
@@ -416,7 +416,7 @@ async fn cargo_toml_component() {
                 .unwrap()
                 .join("wit"),
             revision: 0,
-            wasm_bin_name: None,
```

Comment:
Hmm, could you help me understand the reason for the behavior change here? Is this a bug fix?

Reply:
Yeah I dug through it for a bit and our code does actually try and pull it from the Cargo.toml:

```
// Determine the wasm module name from the [lib] section of Cargo.toml
        let wasm_bin_name = match cargo_toml.lib {
            Some(Product {
                name: Some(lib_name),
                ..
            }) => Some(lib_name),
            _ => None,
        };
```

We were about 7 versions behind on cargo_toml, and it looks like about 8 months ago, there was [a commit](https://gitlab.com/lib.rs/cargo_toml/-/commit/ac1562d58e6f87234cb09541ec06e76fb238e770) that loaded the library name from the package name if it is only a library. So now the behavior is that this name gets set